incubator/hnc/pkg/controllers/object_controller.go

```diff
@@ -50,12 +50,13 @@ type ObjectReconciler struct {
 // +kubebuilder:rbac:groups=*,resources=*,verbs=get;list;watch;create;update;patch;delete;deletecollection;use;impersonate
 
 func (r *ObjectReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
-	if !ex[req.Namespace] {
-		atomic.AddInt32(&obTot, 1)
-		atomic.AddInt32(&obCur, 1)
-		defer atomic.AddInt32(&obCur, -1)
+	if ex[req.Namespace] {
+		return ctrl.Result{}, nil
 	}
 
+	stats.StartObjReconcile(r.GVK.GroupKind())
```

Comment:
Nit: it's sometimes better to make the _caller_ look simpler. So you could say `stats.StartObjReconcile(r.GVK)` and just call `.GroupKind()` inside the function.

There's another school of thought that says "it's better to pass in the minimum amount of information so you don't accidentally come to depend on things (like Version) that you shouldn't care about." In this situation, I'd say that's unlikely to happen so I'd prioritize making the call sites simpler - but it's up to you, this is just a nit.